common/src/main/java/tachyon/conf/TachyonConf.java

```diff
@@ -404,4 +407,17 @@ private String lookupRecursively(final String base, Map<String, String> found) {
     }
     return resolved;
   }
+
+  /**
+   * Constants.USER_FILE_BUFFER_BYTES shuold not bigger than Integer.MAX_VALUE bytes.
```

Comment:
typo: shuold -> should

also, could you write it as
<pre>
{@link Constants.USER_FILE_BUFFER_BYTES} 
</pre>
so under variable rename by IDE, it will be done automatically.